main.js

```diff
@@ -0,0 +1,69 @@
+//2D array representing three pegs on the board
+let startStack = ["5","4","3","2","1"];
+
+var board = {
+  boardState: [
+  ["5", "4", "3", "2", "1"],
+  [],
+  []],
+
+  moveDisc: function (startPeg, endPeg) {
+    

```

Comment:
Watch your indenting and spacing - use prettier if you need to learn good style habits